scripts/helm/publish-chart-yaml.sh

```diff
@@ -49,12 +125,14 @@ branch_chart_version()
     # Develop has no meaningful version
     echo "0.0.0"
   elif [ "$CHECK_BRANCH" == "main" ]; then
-    # Main has no meaningful version, other than the date-time
-    if [ -n "$DATE_TIME" ]; then
-      echo "0.0.0-$DATE_TIME"
-    else
-      echo "0.0.0-main"
+    if [ -z "$LATEST_RELEASE_BRANCH" ]; then
+      LATEST_RELEASE_BRANCH=$(latest_release_branch)
     fi
+    # The main branch helm chart appVersion should follow this format: <bumped-latest>-0-develop-unstable-<timestamp>-0
```

Comment:
Looking at this naming again, should this main rather than develop?

Reply:
Both make sense. The chart is literally the latest (possibly) develop branch chart, we've just given it a unique name and we're pinning it in a registry.  

On the other hand it sits on the main branch.  

IMO, it makes no difference as far as the usage of the chart is concerned. Only thing I can say in favour of 'main' is that it's clear where it comes from, if one were to spot it in the wild.

Your call, I vote 'main'.

Reply:
Changed. Says 'main' now.